### PR TITLE
Add international telephone field

### PR DESCRIPTION
**Pull request by derekcaneja**
Add international telephone field for easily dealing with entering international numbers in meteor.
Use ipinfo.io to pre-populate the dial code input with the dial code of the user's country.

### CODE REVIEW

File: components/phone_input.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+input {
```
Comment (mikedemarais) [inline components/phone_input.css]: Should be `.dial-code, .phone-number { ... }`

File: PhoneFormat.js [hunk 2]
```diff
@@ -1,524 +1,441 @@
...
-phone number: 8646978257
-
-formatE164("US", "8646978257");
- Returns: +18646978257
+Phone = {};
 
-countryForE164Number("US", "+18646978257");
- Returns: US
+// Get the user's current country based on the user's ip address
+Phone._getCountryFromIP = function (callback) {
```
Comment (jperl) [inline PhoneFormat.js]: `ForIp`

File: PhoneFormat.js [hunk 3]
```diff
@@ -1,524 +1,441 @@
...
+  var country = localStorage.getItem('current_country');
```
Comment (jperl) [inline PhoneFormat.js]: Prefix this to not run into other people's local storage. Like `pf.country_code`

File: phone.js [hunk 4]
```diff
@@ -0,0 +1,686 @@
...
+  if (phone.substring(0, 1) == "+") {
+    phone = "+" + phone.replace(/[^\d]/g,'');
+  } else {
+    phone = phone.replace(/[^\d]/g,'');
+  }
+
+  return phone;
+};
+
+var countryCodeToNameArr = new Array();
```
Comment (jperl) [inline phone.js]: Change to object.

**Comment by jperl** [comment]
:+1:

**Pull request merged**